src/Turbo/Twig/FrameTwigExtension.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Turbo\Twig;
+
+use Twig\Environment;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+/**
+ * @author Titouan Galopin <galopintitouan@gmail.com>
+ *
+ * @final
+ * @experimental
+ */
+class FrameTwigExtension extends AbstractExtension
```

Comment:
TurboFrameExtension?

Reply:
I followed the same naming strategy as Webpack Encore Bundle (https://github.com/symfony/webpack-encore-bundle/blob/main/src/Twig/StimulusTwigExtension.php). I quite like having the "TwigExtension" suffix to better understand the difference between DI and Twig extensions in IDEs. 

Adding Turbo as a prefix :)

Reply:
On the other end, none how the bridge's extension have Twig in their suffix:
https://github.com/symfony/symfony/tree/5.x/src/Symfony/Bridge/Twig/Extension
I don't think the suffix is useful personally...